### DISCUSSION TITLE
What kind of community medium do you prefer?

### DISCUSSION THREAD

**Discussion by spacecowboy** [spacecowboy/Feeder#247] [Polls]
There are many ways to let people report bugs or engage in a community. So I'm interesting in what you think

**Comment by vejobo9414larland** [comment]
1. Discord and Matrix have a registration requirement via a phone number. not convenient for everyone. In addition, the advantage in communication speed is not offset by the complexity of information processing
2. forums (as for me) are a fairly convenient form of communication, but you need to raise your server and maintain it....
3. Github doesn’t look bad
4. mail will not be convenient for processing requests and tracking (it seems to me)

**Reply by spacecowboy** [reply]
Nothing in Matrix requires a phonenumber.

The provider you registered with may have required it but I bet you can find one without such a need.

Or you could even self host a server of course. (though now it's real inconvenient!)

**Reply by vejobo9414larland** [reply]
You're right - you don't need a phone number to register in the matrix.
I didn’t use it because most often it comes with discord, and in discord you can’t do it without a phone. So I thought it was the same in the matrix

**Comment by Mr-Bajs** [comment]
Forums should be the main. Hence the topics are clear and if you have an issue or an idea you can easly find if someone else have the same idea, thoughts etc.

Discord is proprietary. Alot me included, just dont' use because of that. Also github are also that but, as the defacto main platform for FOSS development. It's an exeception. And hence the development is done here, no need for a additional registration.

Matrix. It's fine, but it's more a realtime chat, and hence i wouldn't use it for bugreporting. Chats are a mess becouse, issues and topics reoccur here and there and have a tendency to go off-topic if they have any activity. Also required more active moderation.

**Comment by JonDerThan** [comment]
From an automatic email response after reporting a bug:
> Q: Something is different now and I prefer the old way. Can you please change it back? A: It was probably changed for a reason so no, but your feedback is noted.

I'd like you to note that I would very much like you to reenable Github Issues :).

- It's a standardized way of reporting bugs (I had to search like 5 minutes to find a way to report a bug).
- You can easily look up whether the bug you're trying to report was already reported by someone else.
- It makes it way easier to contribute because other people can see and work on the bugs too.

**Comment by smed79** [comment]
Create a telegram chat group.

Example 👇

https://t.me/pDNSf
https://t.me/nekox_broken (linked group)

**Reply by spacecowboy** [reply]
If you want a Telegram group, you can create it and manage it. I will not participate in it.
